bgpd/bgp_vty.c

```diff
@@ -11316,6 +12378,52 @@ static int bgp_show_neighbor_vty(struct vty *vty, const char *name,
 	return CMD_SUCCESS;
 }
 
+
+
+/* "show [ip] bgp neighbors graceful-restart" commands.  */
+DEFUN (show_ip_bgp_neighbors_gracrful_restart,
```

Comment:
can't this be extended to existing show bgp <afi> <safi>?

Reply:
No, This is separate command as this is primarily developed for topotest automation and gives only gr related info not other neighbor info